action.yml

```diff
@@ -192,14 +192,24 @@ runs:
         path: ${{ env.PIP_CACHE_PATH }}
         key: pip-${{ runner.os }}-${{ hashFiles(format('{0}/zephyr/scripts/requirements*.txt', inputs.base-path)) }}
 
-    - name: Install Python packages
+    - name: Install Python packages (Non-Windows)
```

Comment:
let's consider this the "normal" case, just leave it with the "normal" name, hate qualifying things for what they are not, if anything you could call it `(default)` but just `Install Python packages` is good enough.


Or alternatively, could you not implement the different behavior in the command line itself? or you need to use powershell?

Reply:
It's a one-liner with powershell. The `bash` shell on a Windows runners will still have Windows quirks when executing python scripts.